starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,78 @@
 // Names and Input
-
+let hacker1 = "Victoria"
+let hacker2 = prompt("What's the navigator name?");
+console.log("The driver's name is " + hacker1)
+console.log("The navigator name is " + hacker2)
 
 //Conditionals
+function longerName (hacker1, hacker2){
+  if (hacker1.length > hacker2.length) {
+    console.log(`The Driver has the longest name, it has ${hacker1.length} characters`)
+  } else if (hacker1.length < hacker2.length) {
+    console.log(`Yo, navigator got the longest name, it has ${hacker2.length} characters`)
+  } else {
+    console.log(`wow, you both got equally long names, ${hacker1.length} characters!!`)
+  }
+  }
+
+  longerName(hacker1,hacker2);
+
+var capitalizedName = "";
+var reverseName = "";
+for (i = 0; i < hacker1.length; i++ ){
+  capitalizedName += " "+ hacker1[i].toUpperCase();
+}
+console.log(capitalizedName);
```

Comment:
Muy bien, pero si notan está sobrando un espacio en el início de la cadena. Pueden removerlo usando los métodos .slice(1) o .trim().